app/uk/gov/hmrc/pillar2submissionapi/models/overseasreturnnotification/ORNSuccessResponse.scala

```diff
@@ -16,10 +16,51 @@
 
 package uk.gov.hmrc.pillar2submissionapi.models.overseasreturnnotification
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json._
 
-case class ORNSuccessResponse(processingDate: String, formBundleNumber: String)
+case class ORNSuccessResponse(
+  processingDate:       String,
+  accountingPeriodFrom: String,
+  accountingPeriodTo:   String,
+  filedDateGIR:         String,
+  countryGIR:           String,
+  reportingEntityName:  String,
+  TIN:                  String,
+  issuingCountryTIN:    String
```

Comment:
Another question about the parameter types. From a type-safety perspective, could we use custom types instead of using `String`?